docs/schema/concepts/by-theme/transportation/roads.mdx

```diff
@@ -30,7 +30,7 @@ import ExampleTurnRestriction2Connector1 from '!!raw-loader!@site/docs/_examples
 import ExampleTurnRestriction2Connector2 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-connector2.yaml';
```

Comment:
Question: Did we ever get around to documenting one-ways? I think we should do that for transportation GA if we haven't already.

Reply:
there's one sentence that mentions one-ways that is probably insufficient. I'll have to flesh that out later.